executor/executor.go

```diff
@@ -335,29 +335,35 @@ func Next(ctx context.Context, e Executor, req *chunk.Chunk) error {
 	return err
 }
 
-// CancelDDLJobsExec represents a cancel DDL jobs executor.
-type CancelDDLJobsExec struct {
+// CommandDDLJobsExec is the general struct for Cancel/Pause/Resume commands on
+// DDL jobs. These command currently by admin have the very similar struct and
+// operations, it should be a better idea to have them in the same struct
```

Comment:
Please add `.` at the end of a sentence. Please change the other similar places.